### PR TITLE
Add user guide for diet goals

### LINKED ISSUE #62: Add diet goals to UG
To add diet goals to UG as part of v2.0 requirements.

### PR DESCRIPTION
**Pull request by yicheng-toh**
Fixes #62

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -152,6 +152,139 @@ You can list all your diets in AtheltiCLI.
...
+
+You can create one or multiple nutrient goals at once with this command.
+
+
+
+
+**Examples:**
+
+Create multiple nutrients goals:
+* `set-diet-goal calories/500 protein/20 carbs/50 fat/10`
```
Comment (nihalzp) [inline docs/README.md:198]:
```suggestion
* `set-diet-goal calories/500 protein/20 carb/50 fat/10`
```

File: docs/README.md [hunk 2]
```diff
@@ -152,6 +152,139 @@ You can list all your diets in AtheltiCLI.
...
+
+
+You can create one or multiple nutrient goals with this command.
+
+
+**Examples:**
+
+
+Edit multiple nutrients goals:
+* `edit-diet-goal calories/5000 protein/200 carbs/500 fat/100`
```
Comment (nihalzp) [inline docs/README.md:282]:
```suggestion
* `edit-diet-goal calories/5000 protein/200 carb/500 fat/100`
```

**Review by skylee03 — COMMENTED** [review]
LGTM! However, I just realized that we have mistakenly put our user guide in `README.md`, while we are supposed to put it in `UserGuide.md`. Here is a project website example for our reference: https://se-education.org/addressbook-level3/.

**Review by skylee03 — APPROVED** [review]
Maybe we can approve this PR first and put the user guide into the correct file later.

**Pull request merged**